beacon/validator/src/main/java/tech/pegasys/teku/validator/coordinator/ValidatorApiHandler.java

```diff
@@ -636,7 +646,12 @@ public SafeFuture<SendSignedBlockResult> sendSignedBlock(
             maybeBlindedBlockContainer.getSlot());
     return blockPublisher
         .sendSignedBlock(
-            maybeBlindedBlockContainer, broadcastValidationLevel, blockPublishingPerformance)
+            maybeBlindedBlockContainer,
+            // no validation required for locally created blocks
+            isLocallyCreatedBlock(maybeBlindedBlockContainer)
+                ? BroadcastValidationLevel.NOT_REQUIRED
```

Comment:
I'm a bit worried here since we override the request param which could be confusing if for some reason the caller would like to perform the check but ends up not having it. Maybe we could at least log something here?

Reply:
We could add a debug log, but I don't think is really important.

What instead is important is that, thinking more about it, I don't think this is what we want @StefanBratanov .

If the block is produced locally we want just to skip GOSSIP and CONSENSUS but i think that the last EQUIVOCATION step needs to be done if it has been requested (because it doesn't depend on our local behaviour).

Reply:
So I feel like `isLocallyCreatedBlock` should affect `BlockBroadcastValidatorImpl::buildValidationPipeline`

Reply:
I agree, it makes sense. I went for the simpler approach and added a CLI option for enabling this.